torch/csrc/jit/passes/quantization.h

```diff
@@ -22,12 +23,12 @@ TORCH_API void PropagateQuantInfo(std::shared_ptr<Graph>& graph);
  * a tensor.
  *
  * The distribution can then be used for computing qparams for quantization.
- * \param graph is the graph that would be instrumented.
+ * \param module is the module whose methods would be instrumented,
  * \param observer_node is a Node representing a call to observer function. It
  * will be cloned into all the places where we need to add instrumentation.
  */
-TORCH_API void InsertObserverNodes(
-    std::shared_ptr<Graph>& graph,
+TORCH_API std::shared_ptr<script::Module> InsertObserverNodes(
```

Comment:
This API looks like it copies the module, but it is mutating the module. If the intention is to mutate it would be better for this to be `void`.

Reply:
The idea is that it should copy the module, it's just not implemented yet.

I currently have second thoughts about landing this - maybe I'll wait until all module/compilation_unit diffs land and then fix TODO in this one. It might also make sense to allow to specify which method to run on on the API if we don't want observers everywhere.